src/test/java/com/epam/tc/hw2/ex1/Utils.java

```diff
@@ -0,0 +1,56 @@
+package com.epam.tc.hw2.ex1;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Utils {
+    public static final String CHROME_SYSTEM_PROPERTY_NAME = "webdriver.chrome.driver";
+    public static final String CHROME_SYSTEM_PROPERTY_PATH = "/Users/katya/browser_drivers/chromedriver.exe";
+    public static final String HOME_PAGE_TITLE = "Home Page";
+    public static final String HOME_PAGE_URL = "https://jdi-testing.github.io/jdi-light/index.html";
+    public static final String USERNAME = "ROMAN IOVLEV";
+    public static final String LOGIN = "Roman";
+    public static final String PASSWORD = "Jdi1234";
+    public static final int MENU_BUTTONS_NUMBER = 4;
+    public static final List<String> menuButtons = new ArrayList<>(
+        Arrays.asList("HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS"));
+    public static final int IMAGES_NUMBER = 4;
+    public static final List<By> imagesXpathSelectors = new ArrayList<>(
+        Arrays.asList(
+            By.xpath("//span[contains(@class, 'icon-practise')]/../following-sibling::*[@class='benefit-txt']"),
+            By.xpath("//span[contains(@class, 'icon-custom')]/../following-sibling::*[@class='benefit-txt']"),
+            By.xpath("//span[contains(@class, 'icon-multi')]/../following-sibling::*[@class='benefit-txt']"),
+            By.xpath("//span[contains(@class, 'icon-base')]/../following-sibling::*[@class='benefit-txt']"))
+    );
+    public static final int TEXTS_NUMBER = 4;
+    public static final List<String> imagesDescriptions = new ArrayList<>(
+        Arrays.asList(
```

Comment:
`new ArrayList<>(Arrays.asList(` -- is redundant. This can be simplified to `Arrays.asList(` or even `List.of(`